packages/date/src/index.js

```diff
@@ -9,6 +9,10 @@ import 'moment-timezone/moment-timezone-utils';
 
 const WP_ZONE = 'WP';
 
+// This regular expression tests positive for UTC offsets as described in ISO 8601.
+// See: https://en.wikipedia.org/wiki/ISO_8601#Time_offsets_from_UTC
+const VALID_UTC_OFFSET = /^[+-][0-1][0-9](:?[0-9][0-9])?$/;
```

Comment:
Is `+00:99` a valid offset?

Reply:
Probably not. Neither is `+19:00`, but I'm wondering... do we really need to match “valid offsets” only? I mean, I created the (private) regular expression to tell apart strings that “look like” an offset to strings that don’t and, thus, use the appropriate `moment` feature:

- “Oh, this looks like a UTC offset... let’s create a moment instance using `utcOffset`.”
- “Oh, this string is clearly *not* a UTC offset... then I guess the user is using a timezone. Let’s use `tz` to create the moment instance.”

I think we can work under the assumption that the user will provide “valid values” and, when they do, the result is what they’d expect. But if they provide incorrect values... well, the contract is broken and we can’t guarantee anything.

Does this make sense?

Reply:
Fair enough 👍 It could help if the naming or documentation could indicate this, since even if incidentally true for the current circumstances, it could make future maintenance more difficult if one were to assume this is a generally useful UTC offset validation pattern.